internal/wasmldr/README.md

```diff
@@ -0,0 +1,34 @@
+[![Workflow Status](https://github.com/enarx/enarx-wasmldr/workflows/test/badge.svg)](https://github.com/enarx/enarx-wasmldr/actions?query=workflow%3A%22test%22)
+[![Average time to resolve an issue](https://isitmaintained.com/badge/resolution/enarx/enarx-wasmldr.svg)](https://isitmaintained.com/project/enarx/enarx-wasmldr "Average time to resolve an issue")
+[![Percentage of issues still open](https://isitmaintained.com/badge/open/enarx/enarx-wasmldr.svg)](https://isitmaintained.com/project/enarx/enarx-wasmldr "Percentage of issues still open")
```

Comment:
The `cargo readme` lint in `.github/workflows/lint.yml` should be activated for wasmldr also. Along with all the other lints.

Reply:
Fixed in 03bac6c.